src/main/webapp/script.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019 Google LLC
```

Comment:
This file is currently empty. Can you take it out of this PR until there is some content in it?

Reply:
sure, Done :)